docs/visualize_colorspace.md

```diff
@@ -2,12 +2,13 @@
 
 This is a plotting method used to examine all potential colorspaces from available PlantCV functions.
 
-**plantcv.visualize.colorspaces**(*rgb_img*)
+**plantcv.visualize.colorspaces**(*rgb_img, originial_img*)
```

Comment:
and here, is there somewhere else you meant?

Reply:
Ya! Just have to include the default param value too. `originial_img = True*`